tests/conftest.py

```diff
@@ -26,6 +35,7 @@ def homedir():
     '''
 
     tmpdir = tempfile.mkdtemp(prefix='sdc-')
+    configure_locale_and_language()
```

Comment:
This call is redundant. The `homedir` fixture depends on `i18n`, so `configure_locale_and_language` will be called twice. We may also be able to scope `i18n` to `session` since it affects global state. Or maybe just move it into `setUp` so we don't have to manually include it in test cases.